rocket_profile.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, List
+
+from label import TYPE_CHECKING, Label, cp_labels, tantalus_labels
+
+if TYPE_CHECKING:
+    from main import MainApp
+
+
+class RocketProfile:
+    def __init__(self, buttons: Dict[str, str], labels: List[Label]):
+        self.buttons = buttons
+        self.labels = labels
+
+    def update_labels(self, main_window: "MainApp") -> None:
+        for label in self.labels:
+            getattr(main_window, label.name + "Label").setText(
+                label.update(main_window)
+            )
+
+
+tantalus = RocketProfile({"Arm": "arm", "Status": "status"}, tantalus_labels)
+co_pilot = RocketProfile(
+    {"Arm": "arm", "Halo": "halo", "Data": "data", "Status": "status"}, cp_labels
+)
```

Comment:
Could we please move each of these profiles into their own files? In the future, rocket specific behavior might get a lot more complicated, we might have different connections for different rockets, different protocols, maybe we'll have physics models, etc. By having them in separate files, that would allow us to have one directory for each rocket containing all the rocket-specific files.